src/Sulu/Bundle/SecurityBundle/Command/CreateRoleCommand.php

```diff
@@ -81,7 +81,7 @@ function ($value) use (&$securityContextsFlat) {
             $permission = new Permission();
             $permission->setRole($role);
             $permission->setContext($securityContext);
-            $permission->setPermissions(120);
+            $permission->setPermissions(121);
```

Comment:
The permissions are configurable, but this is hard-coded. Makes it possible for random behavior.